### ISSUE TITLE
Ark Server + Mod + Map Mod ?

### ISSUE THREAD

**Issue by FlashModz** [GameServerManagers/LinuxGSM#747]
Hello, you'll add the simple possibility to add mod but also changed the map server easily from the ark?
for ports I have added myself the line port change in the start options for the arkserver not add default?
Thank you for your answers and here are some useful links.

for add mod and map mod : 

https://www.reddit.com/r/playark/comments/3lfjtt/how_to_install_mods_on_ark_private_server/

default launch options ARKSERVER : 

parms="TheIsland?listen"

Me add 

parms="TheIsland?QueryPort=50015?listen"

and the server switch port to function after adding this line

**Comment by FlashModz** [comment]
/serverfiles/ShooterGame/Saved/Config/LinuxServer/GameUserSettings.ini

not change default port launch,

**Comment by UltimateByte** [comment]
We may be able to add a few more start parameters

For example, this dude has : 
`./ShooterGameServer Port=7778?QueryPort=27015?TheIsland?listen?SessionName=[GER]ArkNation|FastXP|FastTaming|?ServerPassword=?ServerAdminPassword=avicularia -server`

http://steamcommunity.com/groups/linuxgsm/discussions/0/523898291501303445/

**Comment by DarthDestroyer** [comment]
I found a similar problem...

I dont know, what happens exactly but it happens after switching to lgsm script.
The game ignores my old (before update) ini and resets it to nothing...
Here is my ini => http://pastebin.com/qp5XEce2

With some start parameters it will at least switch the ports, but it ignores "TheIsland" Parameter and switches to "ShooterEntry" which is nowhere declared!

That is my param Setting => parms="TheIsland?listen?Port=27015?QueryPort=27016?SessionName=[Wurst]Wurst?ServerPassword=?ServerAdminPassword=Wurst?AllowFlyerCarryPVE=True?RCONEnabled=true?RCONPort=32330?RCONServerGameLogBuffer=600?AutoDestroyOldStructuresMultiplier=0?MaxPlayers=127 -nosteamclient -server -battleye"

It has worked before without the Parameters! It is totally creepy...

I tried the ini (link above) with and without the default start params...
With default params, it ignores completly everything and switches to default ports and everything!
